kafka-axon-example/src/main/kotlin/org/axonframework/extension/kafka/example/KafkaAxonExampleApplication.kt

```diff
@@ -152,4 +152,20 @@ class SubscribingConfiguration {
                                          subscribableKafkaMessageSource: SubscribableKafkaMessageSource<String, ByteArray>) {
         eventProcessingConfigurer.registerSubscribingEventProcessor("kafka-group") { subscribableKafkaMessageSource }
     }
+}
+
+/**
+ * If the Consumer Processor Mode is set to Pooled Streaming, that's when we register a
+ * [org.axonframework.eventhandling.pooled.PooledStreamingEventProcessor] using the [StreamableKafkaMessageSource] which the auto
+ * configuration creates.
+ */
+@Configuration
+@ConditionalOnProperty(value = ["axon.kafka.consumer.event-processor-mode"], havingValue = "POOLED_STREAMING")
```

Comment:
Is the `havingValue` case insensitive? I noticed on the config you are using it lower case while here it's upper case.

Reply:
Dunno actually, let me give that a try

Reply:
Nice, it can be lowercase. Gonna change that.